SConstruct

```diff
@@ -19,32 +19,35 @@ sources += Glob("#common/src/main/cpp/export/*.cpp")
 sources += Glob("#common/src/main/cpp/extensions/*.cpp")
 sources += Glob("#common/src/main/cpp/classes/*.cpp")
 
-binary_path = '#demo/addons/godotopenxrvendors/.bin'
+binary_paths = ['#demo/addons/godotopenxrvendors/.bin', '#samples/fb-passthrough-sample/addons/godotopenxrvendors/.bin']
```

Comment:
Rather than looping like this, it probably makes sense to have scons copy the resulting library. Otherwise, you're basically asking scons to do the linking process multiple times (and linking is the slowest part of the build, at least for me).

So, I'd imagine at the end of the `SConstruct` having something like:

```
samples  = ['samples/fb-passthrough-sample']
for sample in samples:
  sample_library = env.Install(target=f'#{sample}/addons/godotopenxrvendors/.bin', source=library)
  Default(sample_library)
```

Then it's only linked once, and copied for each sample.

Reply:
Makes sense, I've applied this change!

Reply:
Looks like the `Install` method wasn't copying over files with their complete filepaths, in latest push I've amended `target` to fix that.